h/models/annotation.py

```diff
@@ -182,8 +173,8 @@ def uuid(self):
     def target_uri(self):
         return self._target_uri
 
-    @target_uri.setter
-    def target_uri(self, value):
+    @target_uri.inplace.setter
+    def _target_uri_setter(self, value):
```

Comment:
https://docs.sqlalchemy.org/en/20/orm/extensions/hybrid.html#using-inplace-to-create-pep-484-compliant-hybrid-properties